### DISCUSSION TITLE
DK starting area can't get rune forging ability

### DISCUSSION THREAD

**Discussion by crprowse** [mod-playerbots/mod-playerbots#525] [General]
When starting the DK the second quest will give you the rune forge ability to complete the third quest. The only way to get this ability is to issue the gm command learn spell. Any help would be great.

**Comment by Dreathean** [comment]
This seems like a general AC question/issue and not specific to playerbots?

**Comment by crprowse** [comment]
Thank you so much for responding,

I not sure, I did build the azerothcore and the DK rune forge worked great. It's just when I build your version. So I'm not sure what I'm missing.

Thanks,
Chris

***@***.***
Christopher Prowse
Server Manager
Little Rock School District
Information Services
***@***.******@***.***>
www.lrsd.org<http://www.lrsd.org/>
501-447-1289
